ast.ts

```diff
@@ -56,3 +58,9 @@ export enum UniOp { Neg, Not };
 export type Value =
     Literal
   | { tag: "object", name: string, address: number}
+
+export type DestructureLHS<A> = { a?: A, lhs: AssignTarget<A>, isStarred : boolean, isIgnore : boolean}
+
+export type AssignTarget<A> = 
```

Comment:
Just making sure I understand, can this design choice in AST accommodate a,b,c = [1,2,3]?

Reply:
The current design will support destructuring a,b,c but since lists are not implemented, it will currently support a,b,c = 1,2,3.

We made a conscious decision to stick to the above cases in order to reduce duplicated efforts. As part of the merging changes, we will extend support to lists, sets, dicts on RHS. The current changes for LHS will by default support them.

We had added all the Milestone 1 working examples in a new design.md file - https://github.com/ucsd-cse231-s22/chocopy-wasm-compiler-B/pull/8/files#diff-7e6258a05da56059e61588bda0ee015fc5486f0f0e9c757c080f0eb005cbd0ba

All the examples listed here are currently working.